Bookshelf/androidApp/proguard-rules.pro

```diff
@@ -0,0 +1,19 @@
+-keepattributes *Annotation*, InnerClasses
+-dontnote kotlinx.serialization.AnnotationsKt # core serialization annotations
+
+# kotlinx-serialization-json specific. Add this if you have java.lang.NoClassDefFoundError kotlinx.serialization.json.JsonObjectSerializer
+-keepclassmembers class kotlinx.serialization.json.** {
+    *** Companion;
+}
+-keepclasseswithmembers class kotlinx.serialization.json.** {
+    kotlinx.serialization.KSerializer serializer(...);
+}
+
+# Change here com.yourcompany.yourpackage
```

Comment:
If people need to manually do something in Proguard for releases, we probably need to document it on the website?

Obviously, it would be best if we can somehow do this for people, but then again, I'm not 100% sure why we need this?

Reply:
This is needed only for `Kotlinx.Serializer` https://github.com/Kotlin/kotlinx.serialization/blob/master/README.md#android the Intro example doesn't use it for instance.